R/PipeOpClassifAvg.R

```diff
@@ -93,15 +85,16 @@ PipeOpClassifAvg = R6Class("PipeOpClassifAvg",
         stop("PipeOpClassifAvg input predictions are incompatible (different levels of target variable).")
       }
 
-      if (has_probs) {
+      prob = NULL
+      if (every(inputs, function(x) !is.null(x$prob))) {
         prob = weighted_matrix_sum(map(inputs, "prob"), weights)
+      } else if (every(inputs, function(x) !is.null(x$response))) {
+        prob = weighted_factor_mean(map(inputs, "response"), weights, lvls)
+      } else {
+        stop("PipeOpClassifAvg input predictions had missing 'prob' and missing 'response' values. At least one of them must be given for all predictions.")
       }
 
-      if (has_classif_response) {
-        response = weighted_factor_mean(map(inputs, "response"), weights, lvls)
-      }
-
-      PredictionClassif$new(row_ids = row_ids, truth = truth, response = response, prob = prob)
```

Comment:
I guess it is on purpose to now always return `prob` and calculate `response` based on this, i.e., the following test should now be fixed? https://github.com/mlr-org/mlr3pipelines/blob/97f38fa14ece48ae887d7bd8e7e1a1d4d7b88386/tests/testthat/test_pipeop_ensemble.R#L89

Reply:
yes, I think this was written when response / prob still worked slightly different in mlr3